Tasks/ServiceFabricDeploy/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 1,
-        "Patch": 2
+        "Patch": 3
```

Comment:
Since this is adding new functionality by exposing new settings, shouldn't this at least be a Minor version change instead of just a Patch?